.github/workflows/test-template.yml

```
@@ -280,7 +294,15 @@ jobs:
           engine-hash: ${{ inputs.engineHash }}
           skip-tsc: false
 
-      - run: pnpm run test:functional --adapter ${{ matrix.adapter }} --shard ${{ matrix.shard }} --engine-type ${{ matrix.engine-type }} --preview-features ${{ matrix.previewFeatures }} --runInBand
+      - name: Get Preview Features
+        id: gpf
+        run: ./.github/workflows/scripts/get-preview-features.js
+        env:
+          PREVIEW_FEATURE_SET: ${{ matrix.previewFeatureSet }}
+          INCLUDED_PREVIEW_FEATURES: driverAdapters
+          EXCLUDED_PREVIEW_FEATURES: ${{ matrix.engine-type == 'wasm' && 'relationJoins' || '' }}
```

```suggestion
         # Exclude relationJoins tests with WASM because the WASM engine does not include the necessary changes yet.
          EXCLUDED_PREVIEW_FEATURES: ${{ matrix.engine-type == 'wasm' && 'relationJoins' || '' }}
```
this was the comment that existed before